app/src/main/java/de/danoeh/antennapod/ui/screen/subscriptions/EpisodeListDefaultSortDialog.java

```diff
@@ -0,0 +1,45 @@
+package de.danoeh.antennapod.ui.screen.subscriptions;
+
+import android.os.Bundle;
+import androidx.annotation.Nullable;
+
+import de.danoeh.antennapod.model.feed.SortOrder;
+import de.danoeh.antennapod.storage.preferences.UserPreferences;
+import de.danoeh.antennapod.ui.screen.feed.ItemSortDialog;
+
+public class EpisodeListDefaultSortDialog extends ItemSortDialog {
+    private static final String ARG_SORT_ORDER = "sortOrder";
+
+    public static EpisodeListDefaultSortDialog newInstance() {
+        Bundle bundle = new Bundle();
+        SortOrder currentSortOrder = UserPreferences.getPrefDefaultSortedOrder();
+        if (currentSortOrder == null) {
+            bundle.putString(ARG_SORT_ORDER, String.valueOf(SortOrder.DATE_NEW_OLD.code));
+        } else {
+            bundle.putString(ARG_SORT_ORDER, String.valueOf(currentSortOrder.code));
+        }
```

Comment:
When accessing the UserPreferences in this class anyway, why not do it directly in `onCreate`? Then it doesn't have to be passed through the `Bundle`.